README.md

```diff
@@ -517,6 +517,15 @@ not supported to write them to individual databases. The first point written
 defines the database, other values are silently ignored!
 
 
+## Contributing
+
+##### Custom influxdb test connection
```

Comment:
It think this should be relocated under the already existing `### Testing` headline. There are already some really basic instructions what is necessary to run the tests. Further information on customizing the test environment probably fit right in.